src/Form/grid/StyledContainer/index.tsx

```diff
@@ -52,6 +62,84 @@ export const StyledContainer = forwardRef<HTMLDivElement, StyledContainerProps>(
 
     useContainerEngine(node, rawNode, ref);
 
+    useEffect(() => {
+      nodeRef.current = node || rawNode;
+    }, [node, rawNode]);
+
+    useEffect(() => {
```

Comment:
should this be pulled out into a hook? like useFixedContainer. maybe the isFixed logic in useContainerStyles could be added to it as well

Reply:
the isFixedContainer function could be put in that file as well. i like the idea of unifying our fixed logic into one place